src/backend/utils/misc/guc.c

```diff
@@ -694,6 +694,12 @@ int			ssl_renegotiation_limit;
 int			huge_pages;
 int			huge_page_size;
 
+/*
+ * OIDs are stored as uint32, so we will add INT_MIN to match the range.
+ */
+int			temp_oid_buffer_start;
+int			temp_oid_buffer_size;
+
```

Comment:
Instead of using these GUCs directly, add two methods to get values which will convert int to uint32 and return